core/src/state/stateful.rs

```diff
@@ -279,6 +279,19 @@ impl<'a, W> StateRef<'a, W> {
     StateRef::new(self.value.inner_ref(), ModifyScope::DATA)
   }
 
+  /// Forget all modifies record in this reference. So the downstream will no
+  /// know the inner value be modified if this reference not be mut accessed
+  /// anymore.
+  pub fn forget_modifies(&self) {
```

Comment:
Does it use 'skip' instead of 'forget' to be better ?

Reply:
seems forget better, means state was really modified. 'skip' feel not modify the state.